packages/tools/src/eventListeners/annotations/contourSegmentation/contourSegmentationCompleted.ts

```diff
@@ -64,13 +66,22 @@ export default async function contourSegmentationCompletedListener(
     return;
   }
 
-  if (!isFreehandContourSegToolRegistered(viewport)) {
+  if (
+    !isFreehandContourSegToolRegistered(
+      viewport,
+      sourceAnnotation.metadata.toolName
```

Comment:
This is probably wrong - but the check down below for handles corresponds to checking if freeform - since that is the actual condition we want it on, I think that one is ok.